invesalius/data/trigger.py

```diff
@@ -33,20 +33,21 @@ def __init__(self, nav_id):
         threading.Thread.__init__(self)
         self.trigger_init = None
         self.stylusplh = False
+        self.COM = False
+        self.nav_id = nav_id
         self.__bind_events()
         try:
             import serial
 
             self.trigger_init = serial.Serial('COM1', baudrate=9600, timeout=0)
-            self.nav_id = nav_id
-            self._pause_ = False
-            self.start()
+            self.COM = True
 
-        except ImportError:
-            print 'PySerial library not installed. Please install to use Trigger option.'
-
-        except serial.serialutil.SerialException:
+        except:
             print 'Connection with port COM1 failed.'
```

Comment:
@rmatsuda, I think you could use a wx.dialog to show this message. Something like this:

wx.MessageBox(_('Connection with port COM1 failed'), _('Communication error'), wx.OK | wx.ICON_ERROR)